packages/react-native-skia/components/image/ImageDataExpiryTime.js

```diff
@@ -0,0 +1,477 @@
+                  alignItems: 'center',
+          }}>
+      <Text
+           style={{ color: '#fff', fontSize: 30,
+                       textAlign: 'center', marginTop: 32,color:'black' }}>
+          Local Image  + Network Image </Text> 
+    </View>
+  </>)
+   }
+    else if(UseCaseCount ==4)
+   {
+       return (
+          <>
+      <View
+          style={{ justifyContent: 'center',
+                   alignItems: 'center',
+                   flexDirection:'row',
+          }}>
+         <View
+             style={{  justifyContent: 'center',
+                       alignItems: 'center',
+                       width: 300,
+                       height: 300,
+                       backgroundColor:'cornsilk',
+         }}>
+            <Image
+                source={require('./disney-1-logo-png-transparent.png')}
+                style={{ width: 200, 
+                         height: 200 ,       
+            }}/>
+        </View>
+        <View
+            style={{ justifyContent: 'center',
+                     alignItems: 'center',
+                     width: 300,
+                     height: 300,
+                     margin:10,
+                     backgroundColor:'cornsilk',
+                     
+            }}>
+            <Image
+                source={{uri: 'http://10.0.2.15:8001/gangster_PNG70.png'}}
+                style={{ width: 200,
+                         height: 200,      
+               }}/>
+        </View>
+    </View>
+    <View 
+      style={{  justifyContent: 'center',
+                alignItems: 'center',
+                flexDirection:'row',
+          }}>
+      <Text
+           style={{ color: '#fff', fontSize:30,
+                       textAlign: 'center', marginTop: 32,color:'black' }}>
+          Local Image  + Network Image no-cache </Text> 
+    </View>
+  </>)
+   }
+   else if(UseCaseCount ==5)
+   {
+       return (
+         <>
+      <View
+          style={{ justifyContent: 'center',
+                   alignItems: 'center',
+                   flexDirection:'row',
+          }}>
+         <View
+             style={{  justifyContent: 'center',
+                       alignItems: 'center',
+                       width: 300,
+                       height: 300,
+                       margin:10,
+                       backgroundColor:'cornsilk',
+         }}>
+            <Image
+                source={require('./disney-1-logo-png-transparent.png')}
+                style={{ width: 200, 
+                         height: 200 ,       
+            }}/>
+        </View>
+        <View
+            style={{ justifyContent: 'center',
+                     alignItems: 'center',
+                     width: 300,
+                     height: 300,
+                     margin:10,
+                     backgroundColor:'cornsilk',
+                     
+            }}>
+            <Image
+                source={{uri: 'http://10.0.2.15:8002/ninja_PNG10.png'}}
```

Comment:
can this server ip and port num be made as variable.
Because if anybody wants to try the app , the server ip will be different . so it will be easy to modify at one place .